parser/parser_test.go

```diff
@@ -4192,6 +4192,16 @@ func TestOptimizerHints(t *testing.T) {
 	require.Len(t, hints, 2)
 	require.Equal(t, "limit_to_cop", hints[0].HintName.L)
 	require.Equal(t, "limit_to_cop", hints[1].HintName.L)
+
+	// Test CTE_INLINE
+	stmt, _, err = p.Parse("with cte(x) as (select /*+ CTE_INLINE(), cte_inline() */ * from t1) select * from cte;", "", "")
```

Comment:
If there is multiple CTEs, do we need to write hint for each CTE to get every CTE inlined?

Reply:
At present, I intend to add hint in each CTE, because hint is added after each “select”. If there is a better way, I will add it later.